src/module.js

```diff
@@ -21,14 +21,17 @@ class QuillPasteSmart extends Clipboard {
 
         let text;
         let html;
+        let file;
 
         if ((!e.clipboardData || !e.clipboardData.getData) &&
            (window.clipboardData && window.clipboardData.getData)) {
             // compatibility with older IE versions
             text = window.clipboardData.getData('Text');
+            file = window.clipboardData.items[0];
```

Comment:
Are you sure about this beeing present in IE?

Reply:
Afaik, prior to IE 11, pasting image was not supported - I'm not sure on which older versions of IE was this aimed for, maybe it would make sense to remove the block altogether.

Reply:
Yes. I think it would be better to remove it.